app/src/main/java/com/kickstarter/ui/adapters/KSListAdapter.kt

```diff
@@ -12,6 +12,7 @@ import com.kickstarter.BuildConfig
 import com.kickstarter.libs.utils.ExceptionUtils
 import com.kickstarter.ui.viewholders.KSViewHolder
 import com.trello.rxlifecycle.ActivityEvent
+import java.util.ArrayList
```

Comment:
Do we need this import?

Reply:
yes, It's explicitly using the ArrayList from java, otherwise the compiler thinks it's the kotlin one, and that will force more code changes within this class or a full migration to kotlin.